agent/README.md

```diff
@@ -7,26 +7,26 @@
 
 ## Usage
 
-Create multiple `AtumContext` with different control measures to be applied 
+Create multiple `AtumContext` with different control measures to be applied
 
 ### Option 1
 ```scala
 val atumContextInstanceWithRecordCount = AtumContext(processor = processor)
-  .withMeasureAdded(RecordCount(MockMeasureNames.recordCount1, measuredColumn = "id"))
+  .withMeasureAdded(RecordCount(MockMeasureNames.recordCount1))
```

Comment:
I am curious why you changed it to `MockMeasureNames`.

Reply:
Possibly IntelliJ refactoring in the past, should be changed though